packages/x-components/src/x-modules/experience-controls/store/__tests__/getters.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { ExperienceControlsRequest } from '@empathyco/x-types';
+import { createExperienceControlsStore, resetExperienceControlsStateWith } from './utils';
+
+describe('testing experience controls module getters', () => {
+  const store = createExperienceControlsStore();
```

Comment:
This test is failing.

```suggestion
  const localVue = createLocalVue();
  localVue.config.productionTip = false;
  localVue.use(Vuex);
  const store = createExperienceControlsStore();
```